modules/priors/vignettes/priors_demo.Rmd

```diff
@@ -294,7 +294,10 @@ vcmax.density <- with(vcmax.dist, pr.dens(distribution, a, b), xlim = c(0,50))
 
 ######### Vcmax Prior Plot
 vcmax.c4 <- all.vcmax.data$ft == 2
-vcmax.data <- transform(all.vcmax.data, mean = Y, se =  1/(sqrt(n) * obs.prec))[,c('ft', 'mean', 'se')]
+vcmax.data <- all.vcmax.data[,c('ft', 'Y')]
+vcmax.data$mean <- vcmax.data$Y
+vcmax.data$se <- 1/(sqrt(n) * obs.prec)
```

Comment:
This looks right if `obs.prec` is a variable in the local environment and not a column in `all.vcmax.data` -- if it's a column then this will ignore it where transform would use it and mask any `obs.prec` from the environment. 

(Apologies if the answer is obvious-- I'm reading this on my phone and didn't pull up the entire file)

Reply:
I checked on a bigger screen and I think `obs.prec` *is* a column in the data frame, so I think it needs to be added to line 297 and then this line needs to become
```suggestion
vcmax.data$se <- 1/(sqrt(vcmax.data$n) * vcmax.data$obs.prec)
```

Reply:
Thanks for pointing that out : ) 
Committed the change.